### ISSUE TITLE
eventDrop defined twice, breaks in IE in strict mode

### ISSUE THREAD

**Issue by gordysc** [icicletech/ember-cli-full-calendar#2]
We are defining the following function twice in ember-cli-full-calendar:

```
eventDrop: function(event, delta, revertFunc, jsEvent, ui, view) {
  _this.sendAction('eventDrop', event, delta, revertFunc, jsEvent, ui, view);
}
```

This causes things to break in IE when using strict mode

**Comment by shrivaraks** [comment]
Hi, thanks! This is been fixed in 0.0.9.

**Comment by gordysc** [comment]
Awesome, thank you!
